cmd/webapp/src/routes/+layout.ts

```diff
@@ -53,10 +53,21 @@ export const load: LayoutLoad = (event) => {
         throw redirect(307, '/claim');
     }
 
+    // Build API client for all child routes (don't throw on invalid for settings/claim pages)
+    const apiClient = createApiClientFromConfig(
+        {
+            endpoint: validated?.endpoint ?? null,
+            apiKey: validated?.apiKey ?? null
+        },
+        fetch,
+        { requireApiKey: false, throwOnInvalid: false }
```

Comment:
@cursoragent do we ever need to pass these arguments now that is centralized call? if not we should clean up the signature